app/assets/javascripts/admin/addon/routes/admin-route-map.js

```diff
@@ -164,8 +164,9 @@ export default function () {
           "adminSearchLogs",
           { path: "/search_logs", resetNamespace: true },
           function () {
+            // eslint-disable-next-line ember/no-unnecessary-index-route
```

Comment:
Based on PR description, sounds like this rule is bogus. So we should just disable globally rather than ignoring for a specific line?

Reply:
Yep, removed.